integrationTests/multiShard/smartContract/testInitilalizer.go

```diff
@@ -0,0 +1,888 @@
+			testNode.accntState = accntAdapter
+			testNode.blkProcessor = blkProcessor
+			testNode.txProcessor = txProcessor
+			testNode.scrForwarder = scrForwarder
+			testNode.blkc = blkc
+			testNode.store = store
+			testNode.txCoordinator = transactionCoordinator
+			testNode.dPool.Headers().RegisterHandler(func(key []byte) {
+				atomic.AddInt32(&testNode.headersRecv, 1)
+				testNode.mutHeaders.Lock()
+				testNode.headersHashes = append(testNode.headersHashes, key)
+				header, _ := testNode.dPool.Headers().Peek(key)
+				testNode.headers = append(testNode.headers, header.(data.HeaderHandler))
+				testNode.mutHeaders.Unlock()
+			})
+			testNode.dPool.MiniBlocks().RegisterHandler(func(key []byte) {
+				atomic.AddInt32(&testNode.miniblocksRecv, 1)
+				testNode.mutMiniblocks.Lock()
+				testNode.miniblocksHashes = append(testNode.miniblocksHashes, key)
+				miniblock, _ := testNode.dPool.MiniBlocks().Peek(key)
+				testNode.miniblocks = append(testNode.miniblocks, miniblock.(*dataBlock.MiniBlock))
+				testNode.mutMiniblocks.Unlock()
+			})
+			testNode.dPool.MetaBlocks().RegisterHandler(func(key []byte) {
+				fmt.Printf("Got metachain header: %v\n", base64.StdEncoding.EncodeToString(key))
+				atomic.AddInt32(&testNode.metachainHdrRecv, 1)
+			})
+			testNode.dPool.Transactions().RegisterHandler(func(key []byte) {
+				atomic.AddInt32(&testNode.txsRecv, 1)
+			})
+			testNode.broadcastMessenger, _ = sposFactory.GetBroadcastMessenger(
+				testMarshalizer,
+				mes,
+				shardCoordinator,
+				sk,
+				&singlesig.SchnorrSigner{},
+			)
+
+			nodes[idx] = testNode
+			idx++
+		}
+	}
+
+	shardCoordinatorMeta, _ := sharding.NewMultiShardCoordinator(uint32(numOfShards), sharding.MetachainShardId)
+	tn := createMetaNetNode(
+		createTestMetaDataPool(),
+		createAccountsDB(),
+		shardCoordinatorMeta,
+		serviceID,
+	)
+	for i := 0; i < numMetaChainNodes; i++ {
+		idx := i + int(numOfShards)*nodesPerShard
+		nodes[idx] = tn
+	}
+
+	return nodes
+}
+
+func getMiniBlocksHashesFromShardIds(body dataBlock.Body, shardIds ...uint32) [][]byte {
+	hashes := make([][]byte, 0)
+
+	for _, miniblock := range body {
+		for _, shardId := range shardIds {
+			if miniblock.ReceiverShardID == shardId {
+				buff, _ := testMarshalizer.Marshal(miniblock)
+				hashes = append(hashes, testHasher.Compute(string(buff)))
+			}
+		}
+	}
+
+	return hashes
+}
+
+func equalSlices(slice1 [][]byte, slice2 [][]byte) bool {
+	if len(slice1) != len(slice2) {
+		return false
+	}
+
+	//check slice1 has all elements in slice2
+	for _, buff1 := range slice1 {
+		found := false
+		for _, buff2 := range slice2 {
+			if bytes.Equal(buff1, buff2) {
+				found = true
+				break
+			}
+		}
+		if !found {
+			return false
+		}
+	}
+
+	//check slice2 has all elements in slice1
+	for _, buff2 := range slice2 {
+		found := false
+		for _, buff1 := range slice1 {
+			if bytes.Equal(buff1, buff2) {
+				found = true
+				break
+			}
+		}
+		if !found {
+			return false
+		}
+	}
+	return true
```

Comment:
add an empty line before.

Reply:
I actually removed this and some other helper functions that were not used anymore